decidim-core/app/assets/javascripts/decidim.js.es6

```diff
@@ -3,11 +3,18 @@
 // = require owl.carousel.min
 // = require svg4everybody.min
 // = require appendAround
+// = require svg-injector
 
 /* globals svg4everybody */
 
 $(document).on('turbolinks:load', () => {
   $(document).foundation();
   $('.js-append').appendAround();
+
+  let externalSvg = $("img.external-svg");
```

Comment:
I recommend using `$externalSvg` as a variable name for jQuery selectors so you can identify them easily.